### ISSUE TITLE
Target link size for pagination

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1256]
Per WCAG AA, the minimum target size link should be 24 pixels (in height and width). Height is fine, but the width for single-digit page numbers is not. I suggest adding 8 pixels padding to the left and right of each page number to increase the target size of the link. I tried it in Figma. 

https://www.figma.com/design/nkgg905s5FyXoyPZjVMco0/EarthWorks?node-id=1642-38952&t=NXbouJPnkfigiSDd-4

![Image](https://github.com/user-attachments/assets/fae54885-ca86-4232-bf9e-6cc4a944c7cd)

Also, I think this was already addressed in another ticket and is currently in review. The order of the paging should be Previous [page numbers] Next. On the screenshot, it's Previous Next [page numbers].
